stream/src/main/scala/libretto/stream/CoreStreams.scala

```diff
@@ -900,10 +900,10 @@ class CoreStreams[DSL <: CoreDSL, Lib <: CoreLib[DSL]](
           .either(upstreamClosed, upstreamValue)
       }
 
-      implicit def positivePolled[A](using A: Junction.Positive[A]): SignalingJunction.Positive[Polled[A]] =
-        SignalingJunction.Positive.eitherPos(
+      given [A : Junction.Positive]: SignalingJunction.Positive[Polled[A]] =
+        SignalingJunction.Positive.eitherPos(using
```

Comment:
Not sure this is just here, or the API needs some love.

Because passing givens explicitly is now "ugly" (which is actually good as one should not normally do it  anyway).

Reply:
It is somewhat questionable, and likely confusing, to have an implicit instance of `SignalingJunction.Positive[Polled[A]]` at all. Maybe it should be reconsidered altogether.

In that light, it is OK to have the implementation somewhat cryptic 😄

Reply:
I have now removed `SignalingJunction.Positive[Polled[A]]` instance.